src/core/matchers/toHaveBeenCalledWith.js

```diff
@@ -44,6 +44,11 @@ getJasmineRequireObj().toHaveBeenCalledWith = function(j$) {
         }
 
         if (matchersUtil.contains(actual.calls.allArgs(), expectedArgs)) {
+          actual.calls
+            .all()
+            .filter(call => matchersUtil.equals(call.args, expectedArgs))
```

Comment:
Any ideas why I can't use matchersUtil.equals here?

Reply:
Each of the failing specs uses a mock `matchersUtil` that doesn't define `equals`. You'll need to update the mocks.

Reply:
Can you point me where they are in the project?